src/components/dashboard-widgets/dashboard-actions-widget/dashboard-actions-widget.component.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { PredefinedWidgetConfig } from '../interfaces/predefined-widget.interface';
+import { ActionConfig, ActionStatus, ActionsWidgetConfig } from '../interfaces/actions-widget.interface';
+
+@Component({
+    selector: 'ux-dashboard-actions-widget',
+    templateUrl: './dashboard-actions-widget.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DashboardActionsWidgetComponent implements PredefinedWidgetConfig, ActionsWidgetConfig {
+    @Input() id: string = '';
+    @Input() name: string = '';
+    @Input() heading: string = '';
+    @Input() fixedMode: boolean = false;
```

Comment:
For boolean and number Inputs we now try to support coercion (supporting string values).

Here is an example of how to update the input: 

https://github.com/UXAspects/UXAspects/blob/develop-3.2.0/src/components/dashboard/widget/dashboard-widget.component.ts#L52

And to support template type checking:

https://github.com/UXAspects/UXAspects/blob/develop-3.2.0/src/components/dashboard/widget/dashboard-widget.component.ts#L319